packages/react-native-reanimated/Common/cpp/Tools/ReanimatedHiddenHeaders.h

```diff
@@ -1,11 +0,0 @@
-#pragma once
-
-#ifdef ANDROID
-#include "Logger.h"
-#include "LoggerInterface.h"
-#include "SpeedChecker.h"
-#else
-#include "Common/cpp/hidden_headers/Logger.h"
-#include "Common/cpp/hidden_headers/LoggerInterface.h"
-#include "Common/cpp/hidden_headers/SpeedChecker.h"
-#endif
```

Comment:
Let's get rid of these hidden headers (i.e. `Logger`, `LoggerInterface` and `SpeedChecker`) as we never use it anyway.

Reply:
As discussed offline, they seem to be used sometimes and IMO it's outside of the scope of this PR.